library/src/amd_detail/hipblaslt-ext.cpp

```diff
@@ -360,8 +669,54 @@ namespace hipblaslt_ext
-        auto rocepilogue    = reinterpret_cast<rocblaslt::RocGemmEpilogue*>(&epilogue);
-        auto rocepinputs    = reinterpret_cast<rocblaslt::RocGemmInputs*>(&gemmInputs);
+        auto            rocepilogue     = reinterpret_cast<rocblaslt::RocGemmEpilogue*>(&epilogue);
+        auto            rocepinputs     = reinterpret_cast<rocblaslt::RocGemmInputs*>(&gemmInputs);
+        auto rocproblemtype = reinterpret_cast<rocblaslt::RocGemmProblemType*>(&gemmProblemType);
+        auto status
+            = RocBlasLtStatusToHIPStatus(rocblaslt_gemm_create_cpp((rocblaslt_handle)m_handle,
+                                                                   m,
+                                                                   n,
+                                                                   batch_count,
+                                                                   k,
+                                                                   lda,
+                                                                   ldb,
+                                                                   ldc,
+                                                                   ldd,
+                                                                   strideA,
+                                                                   strideB,
+                                                                   strideC,
+                                                                   strideD,
+                                                                   *rocepilogue,
+                                                                   *rocepinputs,
+                                                                   *rocproblemtype,
+                                                                   m_data,
+                                                                   m_gemm_count));
+        if(status == HIPBLAS_STATUS_SUCCESS)
+        {
+            m_problem_types[0] = problemtype;
+        }
+        return status;
+    }
+
+    hipblasStatus_t Gemm::setProblem(int64_t          m,
+                                     int64_t          n,
+                                     int64_t          k,
+                                     int64_t          batch_count,
+                                     int64_t          lda,
+                                     int64_t          ldb,
+                                     int64_t          ldc,
+                                     int64_t          ldd,
+                                     int64_t          strideA,
+                                     int64_t          strideB,
+                                     int64_t          strideC,
+                                     int64_t          strideD,
+                                     GemmEpilogueV2&  epilogue,
+                                     GemmInputsV2&    inputs,
+                                     GemmProblemType& problemtype)
```

Comment:
Could we support GemmProblemTypeV2 as well?

Reply:
Will this change in the future? If highly possible, we should change this too.

Reply:
Original setProblem: epilogev1, inputv1, problemtypev
this PR add setProblem: epilogev2, inputv2, problemtypev1
setProblem in the future: epilogev2, inputv2, problemtypev2
Then we will have 3 versions of SetProblem need to take care in the future.
It is not must but better to change it together this time.